src/hooks/useContentView.js

```diff
@@ -0,0 +1,28 @@
+import { useContext } from 'react';
+import { ContentViewContext } from 'contexts/ContentViewContext';
+
+const useContentView = () => {
+  const {
+    displayedComponent,
+    setDisplayedComponent,
+    displayMap,
+    setDisplayMap,
+    navbarColor,
+    setNavbarColor,
+    navbarLeftButton,
+    setNavbarLeftButton,
+  } = useContext(ContentViewContext);
+
+  return {
+    displayedComponent,
+    setDisplayedComponent,
```

Comment:
wdyt about renaming some of these functions to encapsulate their implementation and giving more intuitive usage?
```suggestion
    goTo,
```

Reply:
Done!